game-core/src/main/java/games/strategy/engine/framework/map/download/DownloadFileDescription.java

```diff
@@ -82,7 +82,7 @@ public static DownloadFileDescription ofMapSkinListing(
         .url(skin.getUrl())
         .description(skin.getDescription())
         .mapName(skin.getSkinName())
-        .version(new Version(skin.getVersion()))
+        .version(new Version(skin.getVersion()).getMajor())
```

Comment:
Isn't then skin version the same as any map version?

Reply:
Not as far as I know, skin versioning is independent.
It should become a moot point as the plan is to merge skins into their skinned maps.